arch/s390/slide_hash_vx.c

```
@@ -29,7 +29,13 @@ Z_INTERNAL void slide_hash_vx(deflate_state *s) {
     Assert(s->w_size <= UINT16_MAX, "w_size should fit in uint16_t");
     uint16_t wsize = (uint16_t)s->w_size;
 
-    slide_hash_chain(s->head, HASH_SIZE, wsize);
-    slide_hash_chain(s->prev, wsize, wsize);
+    slide_hash_chain_static(s->head, HASH_SIZE, wsize);
+    slide_hash_chain_static(s->prev, wsize, wsize);
+}
+
+Z_INTERNAL void slide_hash_head_vx(deflate_state *s) {
```

Cppcheck (2.21.0)

[style] The function 'slide_hash_head_vx' is never used.

(unusedFunction)